Makefile

```diff
@@ -1,15 +1,21 @@
 VERSION     := $(shell git describe --always --tags --abbrev=10)
 CMDS         = ls -d cmd/* | xargs -I@ basename @
 CONFIG_PKG   = github.com/soranoba/catfish/pkg/config
+BUILDVCS    := $(shell echo $${BUILDVCS:-true})
+GOARCH      := $(shell echo $${GOARCH:-amd64})
 
 build:
 	cd cmd/catfish/static && npm ci && npm run build
-	${CMDS} | xargs -I@ go build -ldflags "-X ${CONFIG_PKG}.AppVersion=${VERSION}" -o bin/@ ./cmd/@
+	${CMDS} | xargs -I@ go build -ldflags "-X ${CONFIG_PKG}.AppVersion=${VERSION}" -buildvcs=${BUILDVCS} -o bin/@ ./cmd/@
 
-release:
+release: release-js release-app
+
+release-app:
+	${CMDS} | CGO_ENABLED=0 GOOS=linux GOARCH=${GOARCH} xargs -I@ \
```

Comment:
Ah... I'm sorry. Will it be successful to build when deleting the GOARCH and GOOS?

```
build: build-js build-app

build-app:
    ${CMDS} | xargs -I@ go build -ldflags "-X ${CONFIG_PKG}.AppVersion=${VERSION}" -o bin/@ ./cmd/@

build-js:
    cd cmd/catfish/static && npm ci && npm run build
```

I thought this one would be better. Does it work as you expect?

Reply:
@soranoba it's enough. Deleted GOOS and GOARCH.